dptb/postprocess/unified/system.py

```
@@ -188,6 +208,176 @@ def set_atoms(self,struct: Optional[Union[AtomicData, ase.Atoms, str]] = None, o
+                        use_scc: Optional[bool] = None,
+                        nk: Optional[int] = None,
+                        solver: Optional[str] = None) -> Tuple[dict, torch.Tensor]:
+        """Calculate eigenvalues using either bare-model or SCC-corrected H(k)."""
+        if not self._resolve_use_scc(use_scc):
+            data = self._atomic_data if atomic_data is None else atomic_data
+            if k_points is not None:
+                data = self._prepare_kpoint_data(data, k_points)
+            return self.calculator.get_eigenvalues(data, nk=nk, solver=solver)
+
+        hk, sk = self.get_hk(atomic_data=atomic_data, k_points=k_points, use_scc=True)
+        eigvecs, eigvals = self._scc.eigh_solver(
```

Ruff (0.15.12)

[warning] Unpacked variable `eigvecs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)